packages/database/src/core/modelClasses/Question.js

```diff
@@ -1,13 +1,32 @@
+import { SyncDirections } from '@tupaia/constants';
+
 import { MaterializedViewLogDatabaseModel } from '../analytics';
 import { DatabaseRecord } from '../DatabaseRecord';
 import { RECORDS } from '../records';
+import { buildSyncLookupSelect } from '../sync';
 
 export class QuestionRecord extends DatabaseRecord {
   static databaseRecord = RECORDS.QUESTION;
 }
 
 export class QuestionModel extends MaterializedViewLogDatabaseModel {
+  syncDirection = SyncDirections.BIDIRECTIONAL;
+
   get DatabaseRecordClass() {
     return QuestionRecord;
   }
+
+  async buildSyncLookupQueryDetails() {
+    return {
+      select: await buildSyncLookupSelect(this, {
+        projectIds: `ARRAY_AGG(survey.project_id)`,
```

Comment:
This is to aggregate and combine all the project ids into 1 array if there is a question that is being used in multiple surveys from multiple projects